index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portofolio</title>
+</head>
+<body class="Roland">
+    <header class="nav-bar">
+
+        <nav class="nav-list">
+            
+            <a class="logo" href="#">My Logo</a>
+
+            <button class="menu-button">
+                <img class="mobile-menu" src="images/nav.png" alt=" navigation" />
+            </button>
+
+        </nav>
+
+
+    </header>
+
+    <section class="desc-container">
+        <h1 class="text-one">I am Roland Ntwali <br />Glad to see you!</h1>
+        <p class="second-two">
+          I'm a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hestiate to contact me.
+        </p>
```

Comment:
- [ OPTIONAL]  One of the requirements is to make your design identical to that of the Figma template you chose. Currently, this `p` tag should have the following CSS declarations to make it look identical: 

``` CSS
font-family: 'Poppins';
font-style: normal;
font-weight: 400;
font-size: 16px;
line-height: 24px;
color: #344563;
```

#### I think you have the declarations in your CSS stylesheet but the class name you used on the `p` tag is incorrect. 💻